test_tipc/test_train_inference_python_npu.sh

```diff
@@ -18,6 +18,22 @@ sed -i "s/--device:cpu|gpu/--device:cpu|npu/g" $FILENAME
 sed -i "s/--benchmark:True/--benchmark:False/g" $FILENAME
 dataline=`cat $FILENAME`
 
+# parser params
+IFS=$'\n'
+lines=(${dataline})
+modelname=$(func_parser_value "${lines[1]}")
+echo $modelname
+
+if  [ $modelname == "hrnet_w48_contrast" ]; then
+    sed -i "s/lite_train_lite_infer=20/lite_train_lite_infer=10/g" $FILENAME
+    sed -i "s/--save_interval 500/--save_interval 250/g" $FILENAME
+fi
+
+if  [ $modelname == "pfpnnet" ]; then
+    sed -i "s/lite_train_lite_infer=20/lite_train_lite_infer=10/g" $FILENAME
+    sed -i "s/--save_interval 500/--save_interval 250/g" $FILENAME
+fi
+
```

Comment:
hrnet_w48_contrast/pfpnnet needs more than 1h for TIPC running on npu, so the iter is halved here.